src/OrderFormAcceptanceTests.TestData/User.cs

```diff
@@ -0,0 +1,127 @@
+                ConcurrencyStamp = faker.Random.Guid(),
+                PhoneNumber = faker.Phone.PhoneNumber(),
+                PhoneNumberConfirmed = 0,
+                TwoFactorEnabled = 0,
+                LockoutEnd = null,
+                LockoutEnabled = 1,
+                AccessFailedCount = 0,
+                PrimaryOrganisationId = primaryOrganisationId,
+                OrganisationFunction = UserType.ToString(),
+                Disabled = 0,
+                CatalogueAgreementSigned = 0,
+                FirstName = faker.Name.FirstName(),
+                LastName = faker.Name.LastName()
+            };
+        }
+
+        public void Create(string connectionString)
+        {
+            var query = @"INSERT INTO AspNetUsers (
+                            Id,
+                            UserName,
+                            NormalizedUserName,
+                            Email,
+                            NormalizedEmail,
+                            EmailConfirmed,
+                            PasswordHash,
+                            SecurityStamp,
+                            ConcurrencyStamp,
+                            PhoneNumber,
+                            PhoneNumberConfirmed,
+                            TwoFactorEnabled,
+                            LockoutEnd,
+                            LockoutEnabled,
+                            AccessFailedCount,
+                            PrimaryOrganisationId,
+                            OrganisationFunction,
+                            Disabled,
+                            CatalogueAgreementSigned,
+                            FirstName,
+                            LastName
+                        )
+                        VALUES (
+                            @id,
+                            @userName,
+                            @normalizedUserName,
+                            @email,
+                            @normalizedEmail,
+                            @emailConfirmed,
+                            @passwordHash,
+                            @securityStamp,
+                            @concurrencyStamp,
+                            @phoneNumber,
+                            @phoneNumberConfirmed,
+                            @twoFactorEnabled,
+                            @lockoutEnd,
+                            @lockoutEnabled,
+                            @accessFailedCount,
+                            @primaryOrganisationId,
+                            @organisationFunction,
+                            @disabled,
+                            @catalogueAgreementSigned,
+                            @firstName,
+                            @lastName
+                        )";
+
+            SqlExecutor.Execute<User>(connectionString, query, this);
+        }
+        public void Delete(string connectionString)
+        {
+            var query = @"DELETE FROM AspNetUsers WHERE UserName=@userName";
```

Comment:
As above, recommend using the schema name, spaces around operator and terminating semicolon, e.g.

```sql
DELETE FROM dbo.AspNetUsers WHERE UserName = @userName;
```